doc/api/tls.md

```diff
@@ -1914,11 +1919,11 @@ changes:
     decrypted with `object.passphrase` if provided, or `options.passphrase` if
     it is not.
   * `privateKeyEngine` {string} Name of an OpenSSL engine to get private key
-    from. Should be used together with `privateKeyIdentifier`.
+    from. Should be used together with `privateKeyIdentifier`. **Deprecated.**
```

Comment:
nit: IMO the period should go after the bold (but it's not really noticeable anyway)

**Deprecated.** vs **Deprecated**.

Reply:
I went for consistency with the deprecated options from `doc/api/fs.md`, e.g.
https://github.com/nodejs/node/blob/7d14d1fe068dfb34947eb4d328699680a1f5e75d/doc/api/fs.md?plain=1#L1610
https://github.com/nodejs/node/blob/7d14d1fe068dfb34947eb4d328699680a1f5e75d/doc/api/fs.md?plain=1#L4269
https://github.com/nodejs/node/blob/7d14d1fe068dfb34947eb4d328699680a1f5e75d/doc/api/fs.md?plain=1#L1610